apps/web/components/apps/App.tsx

```diff
@@ -441,7 +443,16 @@ const InstallAppButtonChild = ({
         </Button>
       </DropdownMenuTrigger>
       <DropdownMenuPortal>
-        <DropdownMenuContent>
+        <DropdownMenuContent
+        onInteractOutside={(event) => {
+          if (mutation.isLoading) event.preventDefault();
+        }}
+        >
+          {mutation.isLoading && (
+              <div className="z-1 fixed inset-0 flex items-center justify-center bg-black bg-opacity-50">
+                <Spinner />
+              </div>
+            )}
```

Comment:
Adding the div above the dropdown menu should prevent users from clicking additional teams when installing an app and waiting for a response. This also informs the user that their chosen install action is currently being processed.